src/app/examples/segment-example/segment-example.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kirby-segment-example',
+  templateUrl: './segment-example.component.html',
+  styleUrls: ['./segment-example.component.scss'],
+})
+export class SegmentExampleComponent implements OnInit {
+  constructor() {}
+  items: string[] = ['chip-1', 'chip-2', 'chip-3', 'chip-4', 'chip-5', 'chip-6'];
+  selectedItem: string = 'chip-1';
+  onChipSelected(chip: string): void {
```

Comment:
Follow [Angular Style Guide 05-16](https://angular.io/guide/styleguide#style-05-16): "Do name event handler methods with the prefix on followed by the event name."
```suggestion
  onChipSelect(chip: string): void {
```